README.md

```diff
@@ -1,71 +1,86 @@
 [![GitHub build](https://github.com/simplex-chat/simplex-chat/workflows/build/badge.svg)](https://github.com/simplex-chat/simplex-chat/actions?query=workflow%3Abuild)
 [![GitHub release](https://img.shields.io/github/v/release/simplex-chat/simplex-chat)](https://github.com/simplex-chat/simplex-chat/releases)
 
-## Federated chat - private, secure, decentralized
+SimpleX chat objective: to give you the full control of your chat with free open-source protocol and software of simple to deploy servers and simple to use clients providing complete privacy, security and ownership of your contacts and chat.
+
+SimpleX chat prototype is a _thin_ terminal UI on top of [SimpleXMQ](https://github.com/simplex-chat/simplexmq) message broker that uses [SMP protocol](https://github.com/simplex-chat/simplexmq/blob/master/protocol/simplex-messaging.md) and [SMP agent protocol](https://github.com/simplex-chat/simplexmq/blob/master/protocol/agent-protocol.md).
 
 See [simplex.chat](https://simplex.chat) website for chat demo and the explanations of the system and how SMP protocol works.
 
-SimpleX chat prototype is a "thin" terminal UI on top of [SimpleXMQ](https://github.com/simplex-chat/simplexmq) message broker, that uses [SMP protocol](https://github.com/simplex-chat/simplexmq/tree/master/protocol) and SMP agent protocol.
+## Table of contents
+
+- [Features](#features)
+- [Installation](#installation)
+  - [Download executable chat client](#download-executable-chat-client)
+  - [Build from source](#build-from-source)
+    - [with Docker](#with-docker)
+    - [with Haskell stack](#with-haskell-stack)
+- [Usage](#usage)
+  - [Running the chat client](#running-the-chat-client)
+  - [How to use Simplex](#how-to-use-simplex-chat)
+  - [Access chat history](#access-chat-history)
+- [Roadmap](#roadmap)
+- [Disclaimer](#disclaimer)
+- [License](#license)
+
+## Features
 
-These features are implemented:
 - 1-to-1 chat with multiple people in the same terminal window.
-- auto-populated recipient name - just type your messages to reply to the sender.
-- default server is available to use - `smp1.simplex.im:5223` - you can deploy your own server (`smp-server` executable in [simplexmq](https://github.com/simplex-chat/simplexmq) repo).
-- no global identity or any names visible to the server(s) - for the full privacy of your contacts and conversations.
-- E2E encryption, with RSA public key that has to be passed out-of-band (see below).
-- message signing and verification with automatically generated RSA keys.
-- message integrity validation (via including the digests of the previous messages).
-- authentication of each command/message by SMP servers with automatically generated RSA key pairs.
+- Auto-populated recipient name - just type your messages to reply to the sender once the connection is established.
+- Demo SMP server available to use - `smp1.simplex.im:5223`, SMP server executable file is available in [simplexmq](https://github.com/simplex-chat/simplexmq) repo.
```

Comment:
I think `you can deploy` was more .. motivating? Maybe,
```Demo SMP server available to use - `smp1.simplex.im:5223` - you can deploy your own server, see `smp-server` executable in [simplexmq](https://github.com/simplex-chat/simplexmq) repo.```
Not sure, just a suggestion

Reply:
Also, anyway `to use` -> `for use`? And remove `file` after `executable`?

Reply:
Demo SMP server available at...